components/solar-system/spaceSphere.js

```diff
@@ -7,6 +7,7 @@ class SpaceSphere extends React.Component{
     render(){
         return (
             <Sphere
+                lit={this.props.lit}

```

Comment:
What do you think about making all spaceSpheres lit by default since we have the sun lighting the whole scene?

Reply:
Yeah, I know we'd initially chatted about doing that! The only this is it would result in the _sun_ also being lit, which I didn't view as the desired behavior. Thoughts?

Reply:
ah good point, its a "light source" so I think this makes sense.

Reply:
May have misunderstood your meaning (I think we'd talked about setting `lit` to `true` for all spheres no matter what). I'll set a defaultProp value for `lit` so that unless `lit` is set to `false`, it'll be `true`.

Reply:
I think how it is now is fine, this makes sense, keeps it simple.